dvc/remote/local/__init__.py

```diff
@@ -493,21 +498,21 @@ def _unprotect_file(path):
 
         os.chmod(path, os.stat(path).st_mode | stat.S_IWRITE)
 
-    def _unprotect_dir(self, path):
+    def _unprotect_dir(self, path, allow_copy=True):
         for fname in walk_files(path, self.repo.dvcignore):
-            RemoteLOCAL._unprotect_file(fname)
+            RemoteLOCAL._unprotect_file(fname, allow_copy)
```

Comment:
Sorry for picking up these, but while we are at it, we might as well...
```suggestion
            self._unprotect_file(fname, allow_copy)
```

Reply:
done